README.md

```diff
@@ -94,15 +94,19 @@ pip install onnx
 ### Windows
 If you are building ONNX from source on Windows, it is recommended that you also build Protobuf locally as a static library. The version distributed with conda-forge is a DLL and this is a conflict as ONNX expects it to be a static library.
 
+Note that the instructions in this README assume you are using Visual Studio. It is recommended that you run all the commands from a shell started from "Developer Command Prompt for VS 2019" and keep the build system generator for cmake consistent.
```

Comment:
```suggestion
The instructions in this README assume you are using Visual Studio to build ONNX on Windows. It is recommended that you run all the commands from a shell started from "Developer Command Prompt for VS 2019" and keep the build system generator for cmake consistent.
```

A little unclear what you mean by the "build system generator for cmake"

Reply:
Good catch. I will make an example. Thanks!